### PR TITLE
loadgen: test against a repo without .gitpod.yml

### PR DESCRIPTION
**Pull request by jenting**
## Description
Test against a repo without .gitpod.yml

## Related Issue(s)

Fixes https://gitpod.slack.com/archives/C02F19UUW6S/p1664982541052409?thread_ts=1664974807.201539&cid=C02F19UUW6S

## How to test

Run loadgen test

## Release Notes

```release-note
None
```

## Documentation

None

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [x] /werft with-preview
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

File: dev/loadgen/configs/prod-benchmark-pvc.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ## start with
 ##    loadgen benchmark prod-benchmark-pvc.yaml
 
-workspaces: 1
+workspaces: 100
```
Comment (jenting) [inline dev/loadgen/configs/prod-benchmark-pvc.yaml:4]: I change the default number to test 100 workspace with PVC
cc @utam0k

File: dev/loadgen/configs/prod-benchmark-pvc.yaml [hunk 2]
```diff
@@ -84,7 +84,7 @@ repos:
     cloneTarget: main
     score: 20
     workspaceImage: registry.hub.docker.com/gitpod/workspace-full:latest
-  - cloneURL: https://github.com/gitpod-io/template-python-flask
+  - cloneURL: https://github.com/gitpod-io/non-gitpodified-repo
```
Comment (utam0k) [inline dev/loadgen/configs/prod-benchmark-pvc.yaml:87]: Is this related to performance? I ask because I thought that whether this repository is activated should be assured by integration tests and not relates in performance tests.

Comment (jenting) [reply dev/loadgen/configs/prod-benchmark-pvc.yaml:87]: It's not related to performance tests. I agree with your point 👍 .

From my point of view, the loadgen test should contain these workspaces
- stress testing
- regular workspace with prebuild
- regular workspace without prebuild, but with .gitpod.yml
- regular workspace without prebuild and without .gitpod.yml

Comment (utam0k) [reply dev/loadgen/configs/prod-benchmark-pvc.yaml:87]: I'd certainly prefer a repository that makes sense.
